src/Patterns/ProductUpdater.php

```diff
@@ -8,6 +8,44 @@
  * Pattern Images class.
  */
 class ProductUpdater {
+	const DUMMY_PRODUCTS = [
+		[
+			'title'       => 'Vintage Typewriter',
+			'image'       => 'images/pattern-placeholders/watch-hand-brand-jewellery-strap-platinum.jpg',
```

Comment:
It looks like the image here is incorrect: can you take a look & confirm, please? For Vintage Typewriter, I would expect the following image:

<img width="799" alt="Screenshot 2023-12-01 at 16 19 50" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/a945b1d6-f772-4b80-94aa-1bec93bca18c">

Reply:
Of course! Replaced the image here https://github.com/woocommerce/woocommerce-blocks/pull/12007/commits/70a62e2b7811f0240815815fb275e1cfb4aa5892